src/A18NClient/A18nClient.cs

```diff
@@ -0,0 +1,251 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using A18NClient.Dto;
+
+namespace A18NClient
+{
+    public class A18nClient : IA18nClient, IDisposable
+    {
+        private const string BaseUrl = "https://api.a18n.help";
+        private const string RelativePart = "v1/profile";
+
+        private HttpClient _client;
+        private string _defaultProfileId;
+
+        public A18nClient(string apiKey)
+        {
+            _client = new HttpClient
+            {
+                BaseAddress = new Uri(BaseUrl)
+            };
+            _client.DefaultRequestHeaders.Add("x-api-key", apiKey);
+
+        }
+
+        public A18nClient(string apiKey, string profileId) : this(apiKey)
+        {
+            _defaultProfileId = profileId;
+        }
+
+        public void SetDefaultProfileId(string profileId)
+        {
+            _defaultProfileId = profileId;
+        }
+
+        public void Dispose()
+        {
+            if (_client != null)
```

Comment:
Let's implement the `Dispose(bool)` override, as suggested [here](https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose).